.config-schema.json

```diff
@@ -78,7 +78,7 @@
             "type": "array",
             "items": {
                 "type": "string",
-                "enum": ["check-query-cache"]
+                "enum": [""]
```

Comment:
Not sure, if I just remove this or remove whole `"experimentals"` from the config schema

Reply:
Just remove from the enum 👍